README.md

```diff
@@ -1,149 +1,17 @@
 [![Pylint](https://github.com/snu-quiqcl/swift/actions/workflows/pylint.yml/badge.svg)](https://github.com/snu-quiqcl/swift/actions/workflows/pylint.yml)
 
 # swift
-**S**NU **w**idget **i**ntegration **f**ramework for PyQ**t**
+Swift (**S**NU **w**idget **i**ntegration **f**ramework for PyQ**t**) is a framework for integration of PyQt widgets where they can communicate with each other. This project is mainly developed for trapped ion experiment controller GUI in SNU QuIQCL.
 
-A framework for integration of PyQt widgets where they can communicate with each other.
-This project is mainly developed for trapped ion experiment controller GUI in SNU QuIQCL.
+## How to download
+One of the below:
+- git clone or fetch
+- Download the latest release: [v1.0.1](https://github.com/snu-quiqcl/swift/releases/tag/v1.0.1)
```

Comment:
I think this should be separated into two: download and install.
Could you please make an additional instruction of installing the package using `pip`?

Reply:
Oh, I completely forgot it😂

Reply:
I think it is little hard to separate into two, because using released version, we can download and install, simultaneously. So, I just separated in two part: using released version and using git directly.

Reply:
Good decision!